recipes/libcurl/all/conanfile.py

```diff
@@ -467,151 +503,89 @@ def _arm_version(self, arch):
             version = int(match.group(1))
         return version
 
-    def _build_with_autotools(self):
-        with tools.chdir(self._source_subfolder):
-            # autoreconf
-            self.run("{} -fiv".format(tools.get_env("AUTORECONF") or "autoreconf"), win_bash=tools.os_info.is_windows, run_environment=True)
-
-            # fix generated autotools files to have relocatable binaries
-            if tools.is_apple_os(self.settings.os):
-                tools.replace_in_file("configure", "-install_name \\$rpath/", "-install_name @rpath/")
```

Comment:
I didn't migrate this line since I don't know if it is needed with `AutotoolsToolchain` and how to do it.

Reply:
Can be removed if you call https://docs.conan.io/en/latest/reference/conanfile/tools/apple.html#conan-tools-apple-fix-apple-shared-install-name after autotools installation.